src/Dockerfile

```diff
@@ -8,12 +8,12 @@ FROM intel/dlstreamer:2023.0.0-ubuntu22-gpu682-dpcpp as base
 USER root
 
 FROM base as build-default
-COPY ./requirements.txt /requirements.txt
+COPY src/requirements.txt /requirements.txt
```

Comment:
For this to work you'll need to modify the make file build command. Currently it uses "src/" for the build context when it should now use "."

https://github.com/NeethuES-intel/automated-self-checkout/blob/ms-catalog/Makefile#L34
https://github.com/NeethuES-intel/automated-self-checkout/blob/ms-catalog/Makefile#L37